src/main/flight/pid.c

```diff
@@ -638,6 +640,20 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
 
             pidData[axis].D = pidCoefficient[axis].Kd * delta * tpaFactor;
 
+
+            const float pidFeedForward =
+                pidCoefficient[axis].Kd * dynCd * transition *
+                (currentPidSetpoint - previousPidSetpoint[axis]) * tpaFactor / dT;
+            bool addFeedforward = true;
+            if (smartFeedforward)
```

Comment:
Please always use blocks after contitional / loop statements. The nested statements here are a good example of a case where it's very easy to get statements added to the wrong layer without braces.

Reply:
Personally I much prefer terser code which allows viewing more content without scrolling but I'll of course adapt to the project here. Done.

Reply:
Thanks for changing. The terser approach may be fine for an individual project, but in terms of collaboration, an approach where adding an extra statement to a conditional block just adds an extra line, and does not require changes to the surrounding lines to add a block is preferable.